tools/crm_diff.c

```diff
@@ -342,7 +342,7 @@ main(int argc, char **argv)
 
     } else if (options.use_stdin) {
         fprintf(stderr, "Input second XML fragment:");
-        object_2 = stdin2xml();
+        object_1 = pcmk__xml_parse_file(NULL);
```

Comment:
I just want to say it's amazing to me that Coverity flagged this as an error.

```
16:52:33  ===== NEW COVERITY SCAN ERRORS DETECTED =====
16:52:33  Error: COPY_PASTE_ERROR (CWE-398):
16:52:33  /pacemaker/tools/crm_diff.c:334: original: "object_1 = pcmk__xml_parse_file(NULL)" looks like the original copy.
16:52:33  /pacemaker/tools/crm_diff.c:345: copy_paste_error: "object_1" in "object_1 = pcmk__xml_parse_file(NULL)" looks like a copy-paste error.
16:52:33  /pacemaker/tools/crm_diff.c:345: remediation: Should it say "object_2" instead?
16:52:33  ===== END NEW COVERITY SCAN ERRORS DETECTED =====
```

coverity++